mkdocs.yml

```diff
@@ -434,6 +435,8 @@ plugins:
       integrations/sentinelone_deepvisibility.md: xdr/features/collect/integrations/endpoint/sentinelone_deepvisibility.md
       integrations/sophos_edr.md: xdr/features/collect/integrations/endpoint/sophos_edr.md
       integrations/sophos_fw.md: xdr/features/collect/integrations/network/sophos_fw.md
+      integrations/symantec-endpoint-protection.md: xdr/features/collect/integrations/endpoint/symantec_epp.md
+      integrations/symantec_endpoint_protection.md: xdr/features/collect/integrations/endpoint/symantec_epp.md
```

Comment:
```suggestion
```

Reply:
duplicated line

Reply:
The first line use dash ('-') when the second one use underscore ('_')